src/i18n/en-US/index.ts

```diff
@@ -68,7 +68,8 @@ export default {
     blankDestAddress: 'Destination address is blank',
     inputtedInvalidAddress: 'Inputted invalid address',
     selectedInvalidNetworkInWallet: 'Selected invalid network in your wallet',
-    balanceNotEnough: 'You do not have enough ETH in your account to pay for the transaction fee',
+    balanceNotEnough:
```

Comment:
This key is already defined under `warning`. Also, we have a similar message in `error.invalidBalance`. Please reuse.